public/json/printscreen_command+shift+5.json

```
@@ -0,0 +1,23 @@
+{
+  "title": "<Macos> Simple PrintScreen to Command+Shift+5",
+  "rules": [
+    {
+      "description": "Print Screen to Command+Shift+5",
+      "manipulators": [
+        {
+          "type": "basic",
+          "from": {
+            "key_code": "print_screen"
```

```suggestion
            "key_code": "print_screen",
            "modifiers": {
              "optional": [ "caps_lock" ]
            }
```

This is optional; add it if you are okay with this.